onset_maproom/docker/httpd.conf

```diff
@@ -0,0 +1,25 @@
+LoadModule mpm_prefork_module modules/mod_mpm_prefork.so
+LoadModule unixd_module modules/mod_unixd.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule log_config_module modules/mod_log_config.so
+
+User apache
+Group apache
+Listen 8050
+ServerName localhost
+
+ErrorLog "/dev/stderr"
+LogLevel warn
+LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+CustomLog "/dev/stdout" combined
+
+LoadModule wsgi_module /conda/envs/app/lib/python3.9/site-packages/mod_wsgi/server/mod_wsgi-py39.cpython-39-x86_64-linux-gnu.so
+WSGIPythonHome "/conda/envs/app"
+WSGIScriptAlias / /app/docker/app.wsgi
+WSGIDaemonProcess maproom processes=1 threads=10 maximum-requests=1000 python-path=/app
```

Comment:
I remember. It was because of the caching we do with `@lru_cache` in fbfmaproom. The onset maproom doesn't seem to use `@lru_cache`, so maybe we can safely increase the number of processes here. On the other hand, maybe Igor's `@lru_cache` hack would be a quick and dirty way to get acceptable performance on the raster layer (pretty sure that's what it's used for in fbfmaproom2).

Check that out after we get this app published.